src/popup.js

```diff
@@ -16,58 +16,16 @@
  * You should have received a copy of the GNU General Public License
```

Comment:
:+1: yass. I think that @rcutmore started doing some of that work by moving things to htmlUtils.js but obvs this is a WIP